docs/testing.md

```diff
@@ -57,7 +57,7 @@ whatever cluster you have configured.
 
 To run the tests separate from the build go to the `integration_tests` directory and submit
 `run_tests.py` through `spark-submit`.  Be sure to include the necessary jars for the RAPIDS
-plugin either with `spark-submit` or with the cluster when it is [setup](getting-started.md).
+plugin either with `spark-submit` or with the cluster when it is [setup](getting-started-on-prem.md).
```

Comment:
The `.md` was added, but the link is still broken.  Needs to be `get-started/getting-started-on-prem.md`